framework/basehandlers.py

```diff
@@ -148,7 +160,10 @@ def get(self, *args, **kwargs):
 
   def post(self, *args, **kwargs):
     """Handle an incoming HTTP POST request."""
-    self.require_signed_in_and_xsrf_token()
+    is_login_request = str(self.request.url_rule) == '/api/v0/login'
+
+    if not is_login_request:
+      self.require_signed_in_and_xsrf_token()
```

Comment:
@jrobbins, when a user will login using the Google Sign In, the user will send a POST request to backend to verify the Google Sign In token and store it in a session variable. Since, there is no logged in user when this POST request is made, `self.require_signed_in_and_xsrf_token()` will return a 403 response and user will not be able to login. Hence, I have bypassed this method of POST request is made to  `/api/v0/login` which handles login requests.